app/api-requests/claim-service-api.js

```diff
@@ -111,9 +117,8 @@ export const getReviewWithinLast10Months = (dateOfVisit, previousClaims, vetVisi
 }
 
 export const getReviewTestResultWithinLast10Months = (request) => {
-  const { dateOfVisit, previousClaims, latestVetVisitApplication } = getEndemicsClaim(request)
-  const reviewWithinLast10Months = getReviewWithinLast10Months(dateOfVisit, previousClaims, latestVetVisitApplication)
-
+  const { dateOfVisit, previousClaims, latestVetVisitApplication, typeOfLivestock } = getEndemicsClaim(request)
+  const reviewWithinLast10Months = getReviewWithinLast10Months(dateOfVisit, previousClaims, latestVetVisitApplication, typeOfLivestock)
```

Comment:
Could we maybe flatten this, might make it easier to follow?
```suggestion
  const { dateOfVisit, previousClaims, latestVetVisitApplication, typeOfLivestock } = getEndemicsClaim(request)
  const previousClaimsForSpecies = getPastSameSpeciesReviewClaims(previousClaims, dateOfVisit, typeOfLivestock)
  const reviewWithinLast10Months = getReviewWithinLast10Months(dateOfVisit, previousClaimsForSpecies, latestVetVisitApplication)
```

Reply:
looks nicer, but I think we can't move previousClaimsForSpecies from getReviewWithinLast10Months to getReviewTestResultWithinLast10Months as getReviewWithinLast10Months is called from other places

Reply:
fair enough